### ISSUE TITLE
[Monitoring V2]  global registry setting is not passed to the crd chart

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/dashboard#4548]
**Setup**
- Rancher version: 2.6.x
- Browser type & version: chrome Version 95.0.4638.69 (Official Build) (x86_64)

**Describe the bug**

Currently, if I install the rancher-monitoring chart via the UI, it will install two apps: `rancher-monitoring` and `rancher-monitoring-crd`,
Both Apps' `values.yaml` files contain the filed  `global.cattle.systemDefaultRegistry` but UI passes the value to the main chart only 

**To Reproduce**

- Install rancher-monitoring 100.0.0+up16.6.0 via the UI
- choose `Edit YAML` and set `global.cattle.systemDefaultRegistry=rancher-test33`
- track the requests sent out by the UI 
 

**Result**
- UI sends out a PUT request to install the app, and in the payload of the request, we can see the `global.cattle.systemDefaultRegistry`  is set for rancher-monitoring only 
- there is a extra `systemDefaultRegistry` filed at the root level in boh entries
```
{
  "charts": [
    {
      "chartName": "rancher-monitoring-crd",
      "version": "100.0.0+up16.6.0",
      "releaseName": "rancher-monitoring-crd",
      "projectId": "c-zd5w4/p-pk2k7",
      "values": {
        "global": {
          "cattle": {                                  <------------- the field is missing under .global.cattle
            "clusterId": "c-zd5w4",
            "clusterName": "jack-monitoring-test",
            "systemDefaultRegistry": "",
            "url": "https://xxx.xxx.space",
            "rkePathPrefix": "",
            "rkeWindowsPathPrefix": ""
          },
          "systemDefaultRegistry": ""  <-------------- extra one? 
        }
      },
      "annotations": {
        "catalog.cattle.io/ui-source-repo-type": "cluster",
        "catalog.cattle.io/ui-source-repo": "rancher-charts"
      }
    },
    {
      "chartName": "rancher-monitoring",
      "version": "100.0.0+up16.6.0",
      "releaseName": "rancher-monitoring",
      "annotations": {
        "catalog.cattle.io/ui-source-repo-type": "cluster",
        "catalog.cattle.io/ui-source-repo": "rancher-charts"
      },
      "values": {
        "alertmanager": {
          "alertmanagerSpec": {
            "useExistingSecret": true,
            "configSecret": "alertmanager-rancher-monitoring-alertmanager"
          }
        },
        "global": {
          "cattle": {
            "systemDefaultRegistry": "rancher-test33",      <-----------   set 
            "clusterId": "c-zd5w4",
            "clusterName": "jack-monitoring-test",
            "url": "https://xxx.xxx.space",
            "rkePathPrefix": "",
            "rkeWindowsPathPrefix": ""
          },
          "systemDefaultRegistry": ""                             <-------------- extra one? 
        },
        "ingressNginx": {
          "enabled": true
        },
        "prometheus": {
          "prometheusSpec": {
            "evaluationInterval": "1m",
            "retentionSize": "50GiB",
            "scrapeInterval": "1m"
          }
        },
        "rkeControllerManager": {
          "enabled": true
        },
        "rkeEtcd": {
          "enabled": true
        },
        "rkeProxy": {
          "enabled": true
        },
        "rkeScheduler": {
          "enabled": true
        }
      },
      "resetValues": false
    }
  ],
  "noHooks": false,
  "timeout": "600s",
  "wait": true,
  "namespace": "cattle-monitoring-system",
  "projectId": "c-zd5w4/p-pk2k7",
  "force": false,
  "historyMax": 5,
  "cleanupOnFail": false
}
```

**Expected Result**

 the `global.cattle.systemDefaultRegistry`  should be set for both charts

**Comment by ronhorton** [comment]
**Reproduced in 2.6.3**
1. Install monitoring 100.0.0+up16.6.0 via UI
2. Edit yaml to set systemDefaultRegistry: "rancher-test33"
3. Monitored request payload sent 
4. systemDefaultRegistry was sent in rancher-monitoring, but not in rancher-monitoring-crd

**Comment by ronhorton** [comment]
**PASS** Verified fixed in `2.6-head Commit ID 7efbeab`
1. Install monitoring 100.0.0+up16.6.0 via UI
2. Edit yaml to set systemDefaultRegistry: "ZZZZ-TEST-12345"
3. Monitored request payload sent 
4. Value was sent in rancher-monitoring and rancher-monitoring-crd

<img width="395" alt="Screen Shot 2022-01-31 at 12 17 44 PM" src="https://user-images.githubusercontent.com/89212856/151845818-9adf732e-a842-4488-a2ab-a49b136b16b2.png">
<img width="408" alt="Screen Shot 2022-01-31 at 12 17 34 PM" src="https://user-images.githubusercontent.com/89212856/151845903-3dfce6c1-9511-4c57-85bb-2cd41644242a.png">
